01week/rockPaperScissors.js

```diff
@@ -9,9 +9,31 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
-
-  // Write code here
-
+  hand1 = hand1.trim().toLowerCase();
+  hand2 = hand2.trim().toLowerCase();
+  if (hand1 == hand2) {
+    return "It's a tie!";
+  }
+  else if (hand1 == 'rock' && hand2 == 'paper') {
+    return "Hand two wins!";
+  }
+  else if (hand1 == 'paper' && hand2 == 'scissors') {
+    return "Hand two wins!";
+  }
+  else if (hand1 == 'rock' && hand2 == 'scissors') {
+    return "Hand one wins!";
+  }
+  else if (hand1 == 'rock' && hand2 == 'paper') {
+    return "Hand two wins!";
+  }
+  else if (hand1 == 'paper' && hand2 == 'scissors') {
+    return "Hand two wins!";
+  }
+  else if (hand1 == 'rock' && hand2 == 'scissors') {
+    return "Hand one wins!";
+  } else {
+    getPrompt();
+  }
```

Comment:
this logic looks great! but a lot of those conditionals can be shorten/combined! give it a try!